roles/volumes/telemetria-grafana/tasks/main.yaml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Create telemetria-grafana directory if it does not exists
```

Comment:
Sugiro "- name: Telemetria | Create telemetria-grafana directory if it does not exists"